docs/DeveloperGuide.adoc

```diff
@@ -34,24 +34,23 @@ The *_Architecture Diagram_* given above explains the high-level design of the A
 
 [TIP]
 The `.puml` files used to create diagrams in this document can be found in the link:{repoURL}/docs/diagrams/[diagrams] folder.
-Refer to the <<UsingPlantUml#, Using PlantUML guide>> to learn how to create and edit diagrams.
 
-`Main` has two classes called link:{repoURL}/src/main/java/seedu/address/Main.java[`Main`] and link:{repoURL}/src/main/java/seedu/address/MainApp.java[`MainApp`]. It is responsible for,
+`Main` has two classes called link:{repoURL}/src/main/java/seedu/mark/Main.java[`Main`] and link:{repoURL}/src/main/java/seedu/address/MainApp.java[`MainApp`]. It is responsible for,
 
-* At app launch: Initializes the components in the correct sequence, and connects them up with each other.
-* At shut down: Shuts down the components and invokes cleanup method where necessary.
+* Initializes the components in the correct sequence, and connects them up with each other.
+* Shuts down the components and invokes cleanup method where necessary.
 
 <<Design-Commons,*`Commons`*>> represents a collection of classes used by multiple other components.
-The following class plays an important role at the architecture level:
+`LogsCenter` plays an important role at the architecture level. Here is how it works:
 
-* `LogsCenter` : Used by many classes to write log messages to the App's log file.
+* Uses `LogsCenter` to write log messages to the App's log file.
```

Comment:
![image](https://user-images.githubusercontent.com/42093424/67396792-c9461600-f5da-11e9-97c5-5dd7a83d4ab0.png)

I feel it is not necessary to have this as a single bullet here since usually there should be several bullets.

Reply:
Okay